src/encoded/schemas/changelogs/library.md

```diff
@@ -1,5 +1,10 @@
 ## Changelog for library.json
 
+### Schema version 9
+
+* *fragmentation_method* property was replaced by an array *fragmentation_methods*
+* *fragmentation_date* value is specified would apply to all the listed fragmentation methods
```

Comment:
Is specified?

Reply:
changed to "* *fragmentation_date* value, if specified, would apply to all the listed fragmentation methods
"